pages/pipelines/permissions.md.erb

```diff
@@ -1,38 +1,39 @@
+# Controlling user permissions
 
-# Controlling permissions with Teams
-
-Enabling Teams for your organizations gives you fine grained control over the permissions for each of your pipelines.
+Pipeline permissions can be centrally managed by enabling Teams. Enterprise customers have the additional option to set fine-grained user permissions for their organization with the member Permissions page. 
```

Comment:
Would it make sense to add a line here about how it looks if you have neither teams or member permissions?

Reply:
Good idea!